.golangci.yaml

```
@@ -11,17 +13,20 @@
     - govet
     - ineffassign
     - misspell
     - staticcheck
-    - stylecheck
     - unused
+  settings:
+    misspell:
+      locale: US
+    staticcheck:
+      checks:
+        - "all"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)